devices/sinope.js

```diff
@@ -237,6 +237,12 @@ module.exports = [
         vendor: 'Sinope',
         description: 'Zigbee smart dimmer',
         extend: extend.light_onoff_brightness({noConfigure: true}),
+        exposes: [e.light_brightness(), e.effect(), exposes.numeric('led_intensity_on', ea.SET).withValueMin(0).withValueMax(100)
```

Comment:
Instead of adding all exposes here like `e.light_brightness()`, use `[...extend.light_onoff_brightness({noConfigure: true}).exposes, exposes.numeric('led_intensity_off', ea.SET).withValueMin(0).withValueMax(100).withDescription('Control status LED when load OFF')` instead